client/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_CLIENT_ID=998609876632-b0pdipr80lib6jdog6s136tumptg92p2.apps.googleusercontent.com
```

Comment:
You should add `.env` in `.gitignore` and it should not be available on github it contains sensitive details.